app/src/showcase.html

```diff
@@ -0,0 +1,230 @@
+<template>
+  <require from="./theming.css"></require>
+  <section>
+    <h1>Showcase</h1>
+    <p>Find here common usecases and see how each Aurelia UX component fit with the others.</p>
+  </section>
+  <section>
+
+    <ux-card sm="6">
+      <ux-card-header>
+        <span class="ux-card__title">Basic forms exemples</span>
+      </ux-card-header>
+      <ux-card-content>
+        <ux-field>
+          <label>Variant</label>
+          <div class="ux-form__row">
+            <ux-radio model.bind="'filled'" checked.bind="variant"></ux-radio>
+            <label>Filled</label>
+            <ux-radio model.bind="'outline'" checked.bind="variant"></ux-radio>
+            <label>Outline</label>
+          </div>
+        </ux-field>
+      </ux-card-content>
+    </ux-card>
+
+    <ux-grid class="ux-grid--remove-padding">
+      <ux-card sm="6">
+        <ux-card-header><span class="ux-card__title">Labels on top</span></ux-card-header>
+        <ux-card-content>
+          <ux-form>
+            <div class="ux-form__row">
+  
+              <ux-field label="First Name">
+                <ux-input variant.bind="variant" maxlength="10" value.bind="firstName & validate"></ux-input>
+                <ux-input-info ux-input-counter>John</ux-input-info>
+              </ux-field>
+  
+              <ux-field label="Last Name">
+                <ux-input variant.bind="variant" maxlength="10" value.bind="lastName & validate"></ux-input>
+                <ux-input-info ux-input-counter>Doe</ux-input-info>
+              </ux-field>
+  
+            </div>
+  
+            <ux-field label="Email Address">
+              <ux-input variant.bind="variant" type="email" value.bind="email & validate"></ux-input>
+              <ux-input-info>email@example.com</ux-input-info>
+            </ux-field>
+
+            <ux-field label="Contact Type">
+              <ux-select variant.bind="variant" value.bind="contactType" placeholder="Choose an option">
+                <ux-option>Private</ux-option>
+                <ux-option>Professionnal</ux-option>
+              </ux-select>
+            </ux-field>
+  
+            <ux-field label="Biography">
+              <ux-textarea variant.bind="variant" auto-resize></ux-textarea>
+            </ux-field>
+
+            <div class="ux-form__row">
+              <ux-field>
+                <label>Interests</label>
+                <div class="ux-form__row">
+                  <ux-checkbox checked.bind="interests" model.bind="'sport'"></ux-checkbox>
+                  <label>Sport</label>
```

Comment:
Maybe for this section, we should put the checkbox inside the label, so that we can click on the text instead of having to click on the checkbox

Reply:
I agree. In fact we need to decide what standard markup we encourage when using `ux-checkbox`.

So far we have a class `ux-form__row` (in `ux-form` component) available to style elements that must be found on one line. At the beginning I used this class when I wanted checkboxes to be next to each other, but this is not practical for lists of more than 2-3 boxes. Therefore maybe the most conventional way to display checkboxes is to have one per row and the checkbox on the left. By the way material design also allow checkboxes to be hierarchical, but this is out of scope for the component in my opinion.

Suggestion for checkboxes:

```html
<!-- showcase.html -->
<ux-field>
  <label>Interests</label>
  <div class="ux-form__row ux-form__checkbox">
    <label>
      <ux-checkbox></ux-checkbox>
      Sport
    </label>
  </div>
</ux-field>
```

```css
/*  ux-checkbox.css */
.ux-form__checkbox > label {
  display: flex;
  align-items: center;
  align-content: center;
}
```

When thinking about this issue, I suggest to keep in mind that we should have something similar for `ux-switch` but here the idea is that the switch must be displayed on the right. 

```html
<!-- showcase.html -->
<ux-field>
  <label>Notifications</label>
  <div class="ux-form__row ux-form__switch">
    <label>Email Notification</label>
    <ux-switch></ux-switch >
  </div>
</ux-field>
```

```css
/*  ux-switch.css */
.ux-form__switch {
  display: flex;
  align-items: center;
  align-content: center;
  justify-content: space-between;
}
.ux-form__switch > label {
  flex-shrink: 0;
}
```

Markup is not identical but quite close. What are your thoughts about good markup for checkboxes and switches ?